detox/src/devices/drivers/android/genycloud/services/dto/GenyRecipe.js

```diff
@@ -0,0 +1,13 @@
+const _ = require('lodash');
+
+class Recipe {
+  constructor(rawRecipe) {
+    Object.assign(this, _.pick(rawRecipe, 'uuid', 'name'));
```

Comment:
This `_.pick` move is totally not friendly for JS autocompletions. 😢 
Is there a chance you could rewrite to plain `this.uuid =` and `this.name = ` ?

Reply:
sure 👍🏻

Reply:
See #2476